.github/workflows/helm-chart-smoketest.yml

```diff
@@ -108,10 +108,9 @@ jobs:
 
       - name: Create minikube cluster
         if: matrix.config.type == 'minikube'
-        uses: medyagh/setup-minikube@v0.0.19
-        with:
-          container-runtime: containerd
-          kubernetes-version: ${{ env.K8S_VERSION }}
+        run: |
```

Comment:
Is there an upstream (minikube?) issue we'd monitor in anticipation of returning to the GH action in the future?  (Or is this probably a long-term fix?)